salt/docker/init.sls

```diff
@@ -16,6 +16,7 @@ include:
         Environment="NO_PROXY={{ salt['pillar.get']('proxy:no_proxy', '') }}"
   cmd.run:
     - name: systemctl daemon-reload
+    - stateful: True
```

Comment:
In other situations I guess this is the case (still would have to check it), but we have no watchers listening to this `daemon-reload`, no?